src/plugins/data/public/field_formats_provider/field_formats.ts

```diff
@@ -217,10 +223,26 @@ export class FieldFormatRegisty {
   }
 
   register = (fieldFormats: IFieldFormatType[]) => {
-    fieldFormats.forEach(fieldFormat => {
-      this.fieldFormats.set(fieldFormat.id, fieldFormat);
-    });
+    fieldFormats.forEach(fieldFormat => this.fieldFormats.set(fieldFormat.id, fieldFormat));
 
     return this;
   };
+
+  private decorateFieldFormatWithParams = (formatId: IFieldFormatId): Function | undefined => {
```

Comment:
is `Function | undefined` correct? looks like maybe should be `Function | void`?